user/advanced-topics/bind-dirs.md

```diff
@@ -44,9 +44,17 @@ Inside the app qube.
 
 4. Save.
 
-5. Reboot the app qube.
+5. If the directory you are trying to persist doesn't already exist in the app qube, you'll need to create the directory with its full path, under `/rw/bind-dirs`. In this case you would do:
 
-6. Done.
+   ```
+   sudo mkdir -p /rw/bind-dirs/var/lib/tor
+   ```
+   
+   This is required only when this directory does not already exist in the template on which the app qube is based.
```

Comment:
Okay, I just realized that something is still unclear to me. The first clause is:

> If the directory you are trying to persist doesn't already exist in the app qube

But the last sentence (now) is:

> This is required only when this directory does not already exist in the template on which the app qube is based.

So, which is the actual condition?

Is this step required when (a) the directory doesn't exist in the app qube, (b) the directory doesn't exist in the parent template, or (c) both?

Reply:
Both, where (a) is a consequence of (b). Maybe we can suppress the last part. It's just some contextual information as to why that happens. But as you pointed out, it may not be super clear.

If you agree with this, feel free to remove the last part.

Reply:
> Both, where (a) is a consequence of (b).

Okay, so as long as the directory exists in *either* the app qube *or* its template, this step is not required? In other words, this step is required only if the directory does not exist in *either* location?

Are you sure that's correct, or do you mean to say that the direct needs to exist in *both* locations?

Reply:
Sorry for not making myself clear. Generally, if the directory exists in the template, then it will also exist in the app qube because it's is inherited.

This step is not required when the directory already exists in the template (and thus, will already exist in the app qube).

Reply:
Okay, I've attempted to clarify the instructions based on our discussion. How does it look now?

Reply:
Ah, wait, I misunderstood again, but I think I've got it now. Added another commit.

Reply:
Yes, that's it. Thanks!